backend/ee/danswer/background/task_name_builders.py

```diff
@@ -2,5 +2,9 @@ def name_chat_ttl_task(retention_limit_days: int) -> str:
     return f"chat_ttl_{retention_limit_days}_days"
 
```

Comment:
Everything in this module is just refactoring the group and doc permission syncing to be seperate